browsers/node16.14.2-slim-chrome100-ff99-edge/Dockerfile

```diff
@@ -12,6 +12,9 @@ USER root
 
 RUN node --version
 
```

Comment:
Is it ok to edit this Dockerfile since it's already been tagged is in use?

Reply:
I think so? I only pushed the `arm64` build of this new Dockerfile, so currently `amd64` is still on the previous version of this Dockerfile. This Dockerfile is functionally equivalent to the old one too, if you're on `amd64`.